Firmware code/smart_energy_meter/src/token_management.hpp

```diff
@@ -89,12 +91,14 @@ void STS_keypad() {
     dt = 0;
     sts_value = 0;
     delay(20);
-    lcd.clear();
     buss();
+    lcd.clear();
     lcd.setCursor(0, 0);
```

Comment:
I don't know what the `buss()` function does. Could you explain why it is needed for `lcd.clear()` to be called after `buss()` instead of before?

Reply:
Ok I will add comments in the code to explain it

Reply:
The buss() function is there just to emit a sound to signal to the user that he can already enter the token in sts mode